opm/input/eclipse/Schedule/Well/Well.hpp

```diff
@@ -626,6 +636,8 @@ class Well {
         serializer(inj_mult_mode);
         serializer(well_inj_mult);
         serializer(m_filter_concentration);
+        serializer(ref_type);
+        serializer(lgr_tag);
```

Comment:
We generally prefer to list the data members here in the same order as they're declared in the type's list of data members.